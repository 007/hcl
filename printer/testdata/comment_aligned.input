aligned {
   a = "bar" # yoo1
   default = "bar" # yoo2
   bar = "bar" # yoo3
   fatih = ["fatih", "arslan"] // yoo4
   deneme  = {
       bar = "fatih"
   }
   projects = ["vim-go"] # yoo5
   default = "foo" # yoo6
}
