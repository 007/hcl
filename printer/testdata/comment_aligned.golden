aligned = {
	a = "bar"                   # yoo1
	default = "bar"             # yoo2
	bar = "bar"                 # yoo3
	fatih = ["fatih", "arslan"] // yoo4

	deneme = {
		bar = "fatih"
	}
	bar = "bar"                 # yoo3
	fatih = ["fatih", "arslan"] // yoo4

	deneme = {
		bar = "fatih"
	}
}